2024-11-29 20:26:28,404 INFO    MainThread:11708 [wandb_setup.py:_flush():76] Current SDK version is 0.17.4
2024-11-29 20:26:28,404 INFO    MainThread:11708 [wandb_setup.py:_flush():76] Configure stats pid to 11708
2024-11-29 20:26:28,404 INFO    MainThread:11708 [wandb_setup.py:_flush():76] Loading settings from /home/yangxiaobo/.config/wandb/settings
2024-11-29 20:26:28,404 INFO    MainThread:11708 [wandb_setup.py:_flush():76] Loading settings from /data/my_scripts/interesting_toy_exp/multiple_generation_head/wandb/settings
2024-11-29 20:26:28,404 INFO    MainThread:11708 [wandb_setup.py:_flush():76] Loading settings from environment variables: {}
2024-11-29 20:26:28,404 INFO    MainThread:11708 [wandb_setup.py:_flush():76] Applying setup settings: {'_disable_service': False}
2024-11-29 20:26:28,404 INFO    MainThread:11708 [wandb_setup.py:_flush():76] Inferring run settings from compute environment: {'program_relpath': 'multiple_generation_head/train_multiple_softmax.py', 'program_abspath': '/data/my_scripts/interesting_toy_exp/multiple_generation_head/train_multiple_softmax.py', 'program': '/data/my_scripts/interesting_toy_exp/multiple_generation_head/train_multiple_softmax.py'}
2024-11-29 20:26:28,404 INFO    MainThread:11708 [wandb_setup.py:_flush():76] Applying login settings: {}
2024-11-29 20:26:28,404 INFO    MainThread:11708 [wandb_init.py:_log_setup():529] Logging user logs to /data/my_scripts/interesting_toy_exp/multiple_generation_head/wandb/run-20241129_202628-7mc0pvmh/logs/debug.log
2024-11-29 20:26:28,405 INFO    MainThread:11708 [wandb_init.py:_log_setup():530] Logging internal logs to /data/my_scripts/interesting_toy_exp/multiple_generation_head/wandb/run-20241129_202628-7mc0pvmh/logs/debug-internal.log
2024-11-29 20:26:28,405 INFO    MainThread:11708 [wandb_init.py:init():569] calling init triggers
2024-11-29 20:26:28,405 INFO    MainThread:11708 [wandb_init.py:init():576] wandb.init called with sweep_config: {}
config: {'base_model_type': 'gpt2', 'total_batch_size': 524288, 'B': 16, 'T': 1024, 'grad_accum_steps': 4, 'max_lr': 6e-05, 'min_lr': 6e-06, 'warmup_steps': 715, 'max_steps': 19073}
2024-11-29 20:26:28,405 INFO    MainThread:11708 [wandb_init.py:init():619] starting backend
2024-11-29 20:26:28,405 INFO    MainThread:11708 [wandb_init.py:init():623] setting up manager
2024-11-29 20:26:28,405 INFO    MainThread:11708 [backend.py:_multiprocessing_setup():105] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2024-11-29 20:26:28,406 INFO    MainThread:11708 [wandb_init.py:init():631] backend started and connected
2024-11-29 20:26:28,407 INFO    MainThread:11708 [wandb_init.py:init():720] updated telemetry
2024-11-29 20:26:28,411 INFO    MainThread:11708 [wandb_init.py:init():753] communicating run to backend with 90.0 second timeout
2024-11-29 20:26:29,012 INFO    MainThread:11708 [wandb_run.py:_on_init():2402] communicating current version
2024-11-29 20:26:31,211 INFO    MainThread:11708 [wandb_run.py:_on_init():2411] got version response upgrade_message: "wandb version 0.18.7 is available!  To upgrade, please run:\n $ pip install wandb --upgrade"

2024-11-29 20:26:31,211 INFO    MainThread:11708 [wandb_init.py:init():804] starting run threads in backend
2024-11-29 20:26:32,675 WARNING MainThread:11708 [wandb_init.py:init():1185] interrupted
Traceback (most recent call last):
  File "/home/yangxiaobo/miniconda3/envs/test/lib/python3.11/site-packages/wandb/sdk/wandb_init.py", line 1181, in init
    return wi.init()
           ^^^^^^^^^
  File "/home/yangxiaobo/miniconda3/envs/test/lib/python3.11/site-packages/wandb/sdk/wandb_init.py", line 817, in init
    run_start_result = run_start_handle.wait(timeout=30)
                       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/yangxiaobo/miniconda3/envs/test/lib/python3.11/site-packages/wandb/sdk/lib/mailbox.py", line 283, in wait
    found, abandoned = self._slot._get_and_clear(timeout=wait_timeout)
                       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/yangxiaobo/miniconda3/envs/test/lib/python3.11/site-packages/wandb/sdk/lib/mailbox.py", line 130, in _get_and_clear
    if self._wait(timeout=timeout):
       ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/yangxiaobo/miniconda3/envs/test/lib/python3.11/site-packages/wandb/sdk/lib/mailbox.py", line 126, in _wait
    return self._event.wait(timeout=timeout)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/yangxiaobo/miniconda3/envs/test/lib/python3.11/threading.py", line 629, in wait
    signaled = self._cond.wait(timeout)
               ^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/yangxiaobo/miniconda3/envs/test/lib/python3.11/threading.py", line 331, in wait
    gotit = waiter.acquire(True, timeout)
            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
KeyboardInterrupt
2024-11-29 20:26:37,229 WARNING MsgRouterThr:11708 [router.py:message_loop():77] message_loop has been closed
